### ISSUE TITLE
Custom block triggers no-multiple-template-root

### ISSUE THREAD

**Issue by jods4** [vuejs/eslint-plugin-vue#1428]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** not sure, version bundled w/ Vetur?
- **eslint-plugin-vue version:**  same
- **Node version:** 15.3.0
- **Operating System:** Win 10

**Please show your full configuration:**
No config.

**What did you do?**	
I'm trying to use a custom block in SFC.

I did register it in Vetur customBlocks, compiled the grammar and restarted, which now highlights properly.

```vue
<route>
{ title:  "abc" }                             
</route>

<template>
Page A
</template>

<script>
export default {
  setup() {}
}
</script>
```

**What did you expect to happen?**
No eslint error

**What actually happened?**
eslint-plugin-vue reports `vue/no-multiple-template-root`, see screenshot:
![image](https://user-images.githubusercontent.com/3832820/106391570-60109d80-63ee-11eb-93b5-f680d6e851ef.png)

**Comment by ota-meshi** [comment]
Thank you for this issue.
I can't reproduce the problem. Do you get the same error from ESLint or vscode-eslint? If not, it may be a Vetur specific issue.

**Comment by ota-meshi** [comment]
If you have the same problem with the CLI, reopen it.
